### PR TITLE
Product Page

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

First version of the product page.

#### How should this be manually tested?

[Workspace](https://pedro--extensions.myvtex.com/google-shopping/p)

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️| New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: styles/css/home/vtex.store-components.css [hunk 1]
```diff
@@ -20,7 +20,8 @@
   margin-left: auto;
   margin-right: auto;
 }
+
 /* 
 .imageElement--bottomCards { 
   max-height: 33.125rem;
-} */
\ No newline at end of file
+} */
```
Comment (arturpimentel) [inline styles/css/home/vtex.store-components.css]: You may want to remove this comment

File: styles/css/product/vtex.flex-layout.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+.flexRowContent--header {
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+/* .flexRow--reviewRight {
+    align-items: center;
+  } */
```
Comment (arturpimentel) [inline styles/css/product/vtex.flex-layout.css:38]:
```suggestion
```

**Comment by saviomuniz** [comment]
image
O review não tá ocupando todo o width do pai

**Comment by saviomuniz** [comment]
image 
O padding entre esses itens tá diferente (24 x 32)

**Pull request merged**